javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/bytebuddy/AgentCachingPoolStrategy.java

```diff
@@ -61,23 +50,13 @@
    * </ul>
    */
   final Cache<ClassLoader, WeakReference<ClassLoader>> loaderRefCache =
-      CacheBuilder.newBuilder()
-          .weakKeys()
-          .concurrencyLevel(CONCURRENCY_LEVEL)
-          .initialCapacity(LOADER_CAPACITY / 2)
-          .maximumSize(LOADER_CAPACITY)
-          .build();
+      Cache.newBuilder().setWeakKeys().setMaximumSize(LOADER_CAPACITY).build();
```

Comment:
How many class loaders could there possibly be? Does it make sense to limit maximum size here at all?

Reply:
and it's only caching a very small object per ClassLoader, so little downside if there are a lot of class loaders

Reply:
Removed the classloader limit - curious if 64 is too small a number for the other one but leaving it for now.